deepspeech.go

```diff
@@ -63,9 +63,51 @@ type sliceHeader struct {
 // buffer     A 16-bit, mono raw audio signal at the appropriate sample rate.
 // bufferSize The number of samples in the audio signal.
 // sampleRate The sample-rate of the audio signal.
-// TODO Make sure the C string is cleaned properly
 func (m *Model) SpeechToText(buffer []int16, bufferSize, sampleRate uint) string {
-	return C.GoString(C.STT(m.w, (*C.short)(unsafe.Pointer((*sliceHeader)(unsafe.Pointer(&buffer)).Data)), C.uint(bufferSize), C.uint(sampleRate)))
+	str := C.STT(m.w, (*C.short)(unsafe.Pointer((*sliceHeader)(unsafe.Pointer(&buffer)).Data)), C.uint(bufferSize), C.uint(sampleRate));
+	retval := C.GoString(str)
+	C.FreeString(str)
+	return retval
+}
+
+type MetadataItem C.struct_MetadataItem
+
+func (mi *MetadataItem) Character() string {
+	return C.GoString(C.GetCharacter((*C.MetadataItem)(unsafe.Pointer(mi))))
+}
+
+func (mi* MetadataItem) Timestep() int {
+	return int(C.GetTimestep((*C.MetadataItem)(unsafe.Pointer(mi))))
+}
+
+func (mi* MetadataItem) StartTime() float32 {
```

Comment:
Is there any reason why you're using a `float64` instead of `float32` ?

Reply:
I saw `double`s being returned as `float64`, so I figured `float` should be `float32`, but maybe I'm wrong here?

Reply:
You're right, that makes sense